deploy/job_analyze_resource.yml

```diff
@@ -0,0 +1,36 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: noobaa-analyze-resource
+  labels:
+    app: noobaa
+spec:
+  completions: 1
+  parallelism: 1
+  backoffLimit: 1
+  activeDeadlineSeconds: 60
+  ttlSecondsAfterFinished: 10
+  template:
+    spec:
+      volumes:
+      - name: s3-secret
```

Comment:
maybe use a more generic name. `cloud-creds` maybe?

Reply:
I'll change it to `cloud-credentials` (It's below the 253 k8s naming [limit](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#:~:text=This%20means%20the%20name%20must,no%20more%20than%20253%20characters)).